src/recipients/RecipientPickerV2.tsx

```diff
@@ -29,6 +29,7 @@ function RecipientPicker({
     <View style={[styles.body, style]} testID={testID}>
       {title && <Text style={styles.title}>{title}</Text>}
       <FlatList
+        scrollEnabled={false}
```

Comment:
is this related?

Reply:
I saw it in your other PR too... which one does it actually belong with?

Reply:
Should be in #4490.